NuWeather/plugin.py

```diff
@@ -352,18 +380,22 @@ def _apixu_fetcher(self, location):
                           'summary': forecastdata['day']['condition']['text']} for idx, forecastdata in enumerate(data['forecast']['forecastday'])]
         }
 
-    def _darksky_fetcher(self, location):
+    def _darksky_fetcher(self, channel, location):
```

Comment:
Instead of passing channel around, just use the dynamic.msg.args[0] hack Limnoria provides

Reply:
Oh, didn't know that, thanks.